docs/nachet-inference-documentation.md

```diff
@@ -1,12 +1,24 @@
 # Inference Request with n pipelines
 
+## Executive summary
+Following the objective of milestones 1, Nachet back end need to deploy a
```

Comment:
* don't specify management related information in an executive summary
* provide context from business value and user perspective, not technical architecture (no discussion of backend)
* use branding (Nachet Interactive)